cocos/physics-2d/box2d/platform/physics-contact-listener.ts

```diff
@@ -87,17 +95,26 @@ export class PhysicsContactListener extends b2.ContactListener {
     public finalizeContactEvent () {
         PhysicsContactListener._contactMap.forEach((contact: PhysicsContact, key: string) => {
             //console.log('forEach', key, collision);
+
+            // emit collision event
+            if (!contact.disabled || contact.status === Contact2DType.BEGIN_CONTACT) { //BEGIN_CONTACT always emits
```

Comment:
Why always emit event when begin contact.

Reply:
> Why always emit event when begin contact.

PhysicsContact is destroy when ref decrease to 0, and not recycle. User can never get a PhysicsConact before BeginContact to disable it. However this check seems redundant, which I think can be removed

Reply:
It is still not fixed contact.disable in cocos creator 3.7.4 still not working @lealzhan